SITCOMTN-076.tex

```diff
@@ -49,6 +53,325 @@
+
+During the period from installation of LSSTCam on the telescope to the release of Data Preview 2, all communications (including informal discussion) regarding proprietary engineering and on-sky data with LSSTCam are internal to Rubin Project by default.
+Outward-facing communications, including all on-sky LSSTCam images, are reviewed by SIT-Com leadership, the Rubin Celebrations Organizing Committee (RCOC) and its relevant subgroups, and the Communications team, and are approved for release by the Project Director or designated alternate.
+
+Example outward-facing communications:
+
+\begin{itemize}
+
+  \item Weekly digests and News stories on \href{https://rubinobservatory.org/}{Rubin Observatory webpages}
+
+  \item \href{https://www.lsst.org/about/project-status}{Published project status webpage}
+
+  \item Rubin Observatory press releases / media events
+
+  \item Released photographs, plots, and on-sky images
+
+  \item Approved technotes (see Section \ref{technotes})
+
+\end{itemize}
+
+Free and unfettered communication among Project team members is essential for commissioning success.
+%For proprietary data,
+Project team members are required to use only approved Project tools, platforms, and processes for communication, data access and analysis, documentation, software development, work management, etc.
+Derived data products are not subject to those restrictions, but are embargoed until approval for release (see Section \ref{technotes}).
+In practice, we expect most work done by the Project team on the commissioning data to be done within protected directories at the Rubin US Data Facility at SLAC.
+
+Prior to the release of associated data products as part of the Early Science program, derived data products from LSSTCam on-sky commissioning data may be shared beyond the Project team only in the following situations:
+
+\begin{enumerate}
+
+  \item the derived data product has been approved for release in one of the official Rubin Observatory outward-facing communication channels;
+
+  \item the derived data product is documented as part of an approved technote (see Section \ref{technotes}).
+
+\end{enumerate}
+
+Derived data products that represent visit, coadd, and difference images from LSSTCam on-sky commissioning, with the exception of alert postage stamp images, are embargoed for a period of at least 30 days following the observation.
+%not corresponding to artificial Earth-orbiting satellites
+
+%Derived data products based on commissioning on-sky data will not be released for at least 80 hours following the observation.
+
+During the on-sky commissioning period with LSSTCam, members of the Project team are allowed to discuss technical details of their work outside the team, and they may freely discuss aspects that do NOT relate to specific on-sky data products from LSSTCam or interim science performance.
```

Comment:
This is tricky.  Purely verbal communication are simple, but in practice most of these discussions are likely to revolve around figures, involving Rubin data.   Do we have a way of bowdlerising the data that's acceptable?   Is someone allowed to show histograms of proper motion errors if there's no pointing attached?  Seeing histograms?

Reply:
This sentence is meant to cover general discussions of work assignments, e.g., "I'm working on algorithm XYZ to improve astrometric performance"

Sharing plots representing on-sky data would be covered under the derived data products that could be shared via technotes